mobile/library/cc/log_level.h

```diff
@@ -7,7 +7,15 @@
 namespace Envoy {
 namespace Platform {
 
-using LogLevel = Envoy::Logger::Logger::Levels;
+enum class LogLevel {
+  trace = Envoy::Logger::Logger::Levels::trace,
```

Comment:
Should we switch these to PascalCase (Trace, Debug, etc) as per the style guide, now that we get to chose the names? https://github.com/envoyproxy/envoy/blob/main/STYLE.md

Reply:
I'm wondering what's best here: following the style guide or staying consistent with the "main" Envoy log levels 🤔 

I'm inclined to keep the scope smaller here. What do you think?

Reply:
I *think* that if we're going through the effort to make our own enum to use in this case which can not be implicitly converted to the existing spdlog level, I'd be inclined to go all the way. That being said, I don't feel at all strongly about this and we could even to that in a followup if we wished. So the current PR LGTM.